Makefile

```diff
@@ -420,7 +420,12 @@ tests_coverage:
 	@$(MAKE) clean
 	@$(MAKE) --no-print-directory tests PX4_CMAKE_BUILD_TYPE=Coverage
 	@mkdir -p coverage
-	@lcov --directory build/px4_sitl_test --base-directory build/px4_sitl_test --gcov-tool gcov --capture -o coverage/lcov.info
+	@lcov --directory build/px4_sitl_test \
+		--base-directory build/px4_sitl_test \
+		--gcov-tool gcov \
+		--capture \
+		--ignore-errors mismatch \
```

Comment:
Can you point me to the error?
It is generally helpful to have this in the commit message.

Reply:
Agreed I'll make sure add the error next time, for now here's a link to the failed CI run 

https://github.com/PX4/PX4-Autopilot/actions/runs/15715755395/job/44284939514

Reply:
Thanks, based on https://github.com/linux-test-project/lcov/issues/316 it seems to be a bug. I would add a comment, so it can be removed again in future.